apps/researcher/src/lib/community.test.ts

```diff
@@ -1,6 +1,40 @@
 import {describe, expect, it} from '@jest/globals';
 import {auth} from '@clerk/nextjs';
-import {isAdmin, Membership, sort, SortBy} from './community';
+import {isAdmin, sort, SortBy} from './community';
+import {OrganizationMembership} from '@clerk/nextjs/dist/types/server';
+
+const basicOrganization = {
+  id: 'organization1',
+  name: 'Organization 1',
+  slug: 'organization-1',
+  imageUrl: 'https://example.com/image.png',
+  createdAt: 1690000000000,
+  updatedAt: 1690000000000,
+  logoUrl: null,
```

Comment:
To be sure: this should indeed be `null`, not `undefined`?

Reply:
No, logo is `string | null`